.github/actions/install-dependencies/action.yml

```diff
@@ -22,4 +22,4 @@ runs:
       if [ "$TARGET_ARCH" = "x86" ]; then
         export npm_config_arch="ia32"
       fi
-      node script/yarn.js install --immutable
+      node script/yarn.js install --immutable-cache
```

Comment:
I'm fine with the change here, but am asking so I'll know if there are followup tasks for after this PR: does this need to be backported to the maintenance branches?